packages/design-system/src/styles/components/_Button.scss

```diff
@@ -1,5 +1,10 @@
 @import '../settings/index.scss';
 
+// Danger button is marked for deprecation //
+.ds-c-button--danger {
+  --deprecated: 'WARNING: Use .ds-c-button or .ds-c-button--primary instead';
+}
```

Comment:
Just curious, would I be able to see this `deprecated` message on the browser? Or it's placed here so coders are aware of the deprecation?

Reply:
This will show in the code - it was a way to show it in the CSS side of things 👍